src/bot/commands/helpers/checkElevatedRights.ts

```diff
@@ -0,0 +1,9 @@
+import Config from '@config/Config';
+import { Collection, Role } from 'discord.js';
+
+const userHasElevatedRole = (rolesCache: Collection<string, Role>) => {
+    const config = new Config();
```

Comment:
You can directly do `import { config } from '@util/Container';` here, so there is no need to instantiate. In fact instantiating a separate config, will lead to some problems, so it's important to use the same one everywhere. This is one of the quirks of the project setup, sorry.

Reply:
It makes sense, no problem.